src/rred_reports/reports/emails.py

```diff
@@ -156,6 +155,7 @@ def send_email(message: Message, save: bool = False) -> bool:
 
         Args:
             message (Message): Constructed message to send
+            save (bool): Should the email be saved to the sent mailbox
```

Comment:
I think this was set to default to `False` for testing, but we might want it to default to `True` in prod?

Reply:
yeah good call there